main.html

```diff
@@ -6,60 +6,75 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     
     <title>Document</title>
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel=”stylesheet” 
+    href=”https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css” />
+    <link rel="stylesheet" 
+    href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" 
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" 
+    crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@200&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="row no-gutters">
-        <div class="col-md-4 no gutters">
-            <div class="left d-flex justify-content-center align-items-center">
-                <h2>comproDLS</h2>
-                <h1>OnBoarding</h1>
-                <h3>Application</h2>
-                <p>
-                    Welcome to comproDLS OnBoarding Application for mentors and freshers.
-                    Please sign in using your organization email and password.
-
-                </p>
-                <p id = "second">
-                    In case you do not have an account ,click on "Sign up now" to get started.
-                </p>
+  <div class="row  no-gutters">
+    <div class="col-md-12  col-lg-4">
+      <div class="left  d-flex">
```

Comment:
Why do we still need `d-flex` class? I can't see any CSS styles define for d-flex?